src/main/java/com/sprint/mission/discodeit/controller/UserController.java

```diff
@@ -0,0 +1,97 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.request.BinaryContentCreateRequest;
+import com.sprint.mission.discodeit.dto.request.UserCreateRequest;
+import com.sprint.mission.discodeit.dto.request.UserStatusUpdateRequest;
+import com.sprint.mission.discodeit.dto.request.UserUpdateRequest;
+import com.sprint.mission.discodeit.dto.request.UserUpdateRequestDto;
+import com.sprint.mission.discodeit.service.UserService;
+import com.sprint.mission.discodeit.service.UserStatusService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/users")
+public class UserController {
+
+    private final UserService userService;
+    private final UserStatusService userStatusService;
+
+    @PostMapping()
+    public ResponseEntity<Object> userRegister(@RequestBody UserCreateRequest userCreateRequest,
+        @RequestBody Optional<BinaryContentCreateRequest> profileCreateRequest) {
```

Comment:
`@RequestBody`대신 `@RequestPart`를 사용해야 프로필 이미지 등록할 때 정상동작할 것 같아요.
```suggestion
    public ResponseEntity<Object> userRegister(@RequestPart UserCreateRequest userCreateRequest,
        @RequestPart(required = false) Optional<BinaryContentCreateRequest> profileCreateRequest) {
```

지금은 아래와 같은 에러가 뜨더라구요.
```
Content-Type 'multipart/form-data;boundary=----WebKitFormBoundaryDFCWr4tmrMnFraXP;charset=UTF-8' is not supported 
```